packages/scripts/config/webpack.config.js

```diff
@@ -103,7 +103,7 @@ const baseConfig = {
 	target,
 	output: {
 		filename: '[name].js',
-		chunkFilename: '[name].js?v=[chunkhash]',
+		chunkFilename: '[name].js?ver=[chunkhash]',
```

Comment:
WordPress adds `?ver=` to all urls. I updated it for consistency. It doesn't matter in practice.